packages/block-library/src/image/image-editing/use-transform-image.js

```diff
@@ -90,6 +101,9 @@ function useTransformState( { url, naturalWidth, naturalHeight } ) {
 		const el = new window.Image();
 		el.src = url;
 		el.onload = editImage;
+		if ( imageEditingCrossOriginValue ) {
```

Comment:
We can't rely on a falsy check, `""` empty string is a valid value.

```suggestion
		if ( typeof imageEditingCrossOriginValue === 'string' ) {
```

Reply:
👍🏻